Say Their Names/Source/Controller/Petitions/PetitionsController.swift

```diff
@@ -8,8 +8,21 @@
 
 import UIKit
 
-class PetitionsController: BaseViewController {
-    override func viewDidLoad() {
-        super.viewDidLoad()
+/// Controller responsible for showing the petitions
+final class PetitionsController: BaseViewController {
+    
+    private let ui: PetitionsUI
```

Comment:
We should avoid abbreviations and raw variables. Here, the best name is `petitionsUI` or maybe `viewCode`.
Also, we might want to make it lazy var

Reply:
another idea is to have something  `private let contentView: PetitionsView` to be clear with the purpose of the variable

Reply:
I chose `ui` as it could be used as a universal member across all the different views we will have. I think the next best thing would be `petitionsView` as it is more clear.